migration/i8.2z/oracle/202106281200_IDEMPIERE-4863.sql

```diff
@@ -0,0 +1,94 @@
+-- Table: ad_wlistbox_customization
+
+CREATE TABLE ad_wlistbox_customization
+(
+    ad_client_id 					NUMBER(10) NOT NULL,
+    ad_org_id 						NUMBER(10) NOT NULL,
+    ad_wlistbox_customization_id 	NUMBER(10) NOT NULL, 
+    wlistboxname 					VARCHAR(60) NOT NULL,
```

Comment:
how do you make sure wlistboxname is unique ? at minimum, you will need a unique constraint for it but that's still hard to make sure it is unique and consistent (naming convention wise) across the applications.